core/tests/test_apps.common.cbor.py

```diff
@@ -70,5 +75,31 @@ def test_cbor_encoding(self):
             self.assertEqual(unhexlify(encoded), encode(val))
             self.assertEqual(val, decode(unhexlify(encoded)))
 
+    
+    def test_cbor_only_encoding(self):
```

Comment:
These tests can't be added to the previous function, because decoding doesn't yield the same object, since duplicates are ignored.

Reply:
The name `test_cbor_only_encoding` is not clear. Let's use something longer, but more descriptive, e.g. `test_cbor_sets_with_duplicates`

Reply:
I named it this way, because the `test_cbor_encoding` is also pretty generic and I wanted to stick to that convention here. But since it is not generic at the moment, I can rename it.